CMakeLists.txt

```diff
@@ -70,7 +70,7 @@ set(GS_SOURCES
     ${PROJECT_SOURCE_DIR}/src/requests/rejectpayrequest.cpp
     ${PROJECT_SOURCE_DIR}/src/requests/getinforequest.cpp
     ${PROJECT_SOURCE_DIR}/src/requests/sendrawtxrequest.cpp
-    ${PROJECT_SOURCE_DIR}/src/routers_http.cpp
```

Comment:
why is that?

Reply:
Not sure I understand the question. There was duplication of ${PROJECT_SOURCE_DIR}/src/routers_http.cpp

Reply:
ok